Web/src/components/games/PandaSequence/GameDisplay.tsx

```diff
@@ -57,19 +68,45 @@ const Score = styled.h3`
   padding-top: 6px;
 `;
 
+/**
+ * This component handles both display and input mode.
+ * Display Mode: Element in sequence is animated according to "displaying" prop
+ * Input Mode: Element tapped is animated based on "selected" state
+ */
 const GameDisplay: React.FC<IProps> = ({
-  mode, secondsLeft, score, active, handleInputEvent,
+  mode, secondsLeft, score, processInput, timestep, displaying,
 }) => {
-  const onTap = (index:number) => {
-    // TODO: Animate tapped component to confirm user input
-    handleInputEvent(index);
+  const [selected, setSelected] = useState(Array(5).fill(false));
```

Comment:
Can there be more than one that's selected (such that you're creating a bitmap)?

Otherwise, you just need to track the index of the thing currently being selected right? Something like `useState<number | null>(null);`

Reply:
Yes there could be! Because a user might tap on a second object before the first object is released, and we need to respond to that second action simultaneously :)

Reply:
Cool!